lib/__tests__/package-integrity/handler/integrity.test.ts

```diff
@@ -0,0 +1,210 @@
+import * as child from 'child_process';
+import * as os from 'os';
+import * as path from 'path';
+import AdmZip from 'adm-zip';
+import * as fs from 'fs-extra';
+import * as tar from 'tar';
+import { NpmArtifactIntegrity, PublishedPackage, PyPIArtifactIntegrity, RepositoryIntegrity } from '../../../package-integrity/handler/integrity';
+import { Repository } from '../../../package-integrity/handler/repository';
+
+function fixturePath(name: string) {
+  return path.join(__dirname, '__fixtures__', name);
+}
+
+type Download = (pkg: PublishedPackage, target: string) => void;
+
+interface IntegrityInputs {
+  npmDownload: Download;
+  pypiDownload: Download;
+  repoDir: string;
+}
+
+function initializeRepo(repoDir: string): Repository {
+
+  const shell = (command: string) => child.execSync(command, { cwd: repoDir, stdio: ['ignore', 'inherit', 'inherit'] });
```

Comment:
> Also, I personally prefer exec forms that take arguments as an array of strings to look safer

I kind of share the sentiment, but I also really like using the full command form since it allows easy copy-pasting of commands from code to terminal. It's also easier to call 

> Probably should slap shell: true in there to be Windows friendly...

Unless i'm missing something, `execSync` defaults to using an appropriate shell per platform. See [execSync](https://nodejs.org/api/child_process.html#child_processexecsynccommand-options).